ext/bg/js/backend.js

```diff
@@ -101,6 +102,10 @@ class Backend {
         }
     }
 
+    _onClipboardText(text) {
+        this._onCommandSearch({mode: 'popup', query: text});
+    }
+
```

Comment:
Not sure if this is a good way to call the search command

Reply:
Could make a utility function(s) for the various search functionalities if you want to avoid extra (de)structuring, but I don't think it's a big concern; looks fine as-is to me